app/Repositories/ModuleRepository.php

```diff
@@ -66,4 +68,23 @@ public function prepareFieldsBeforeSave(TwillModelContract $object, array $field
 
         return parent::prepareFieldsBeforeSave($object, $fields);
     }
+
+    /**
+     * Dupes Twill PR https://github.com/area17/twill/pull/2763.
+     * This can be deleted once that is merges and we update
+     */
+    public function cmsSearch(string $search, array $fields = [], callable $query = null): Collection
+    {
+        $searchFilter = new FreeTextSearch();
+        $searchFilter->queryString($search);
+        $searchFilter->searchFor($search);
+        $searchFilter->searchColumns($fields);
+        $searchFilter->searchQuery($query);
+
+        $builder = $this->model->latest();
+
+        $searchFilter->applyFilter($builder);
+
+        return $builder->get();
+    }
```

Comment:
Could we make this a patch instead?

Reply:
Yea, that's a good call. So when the PR gets accepted the patch will automatically be obsolete.